### PR TITLE
Doc 783 improve list function explanations

### PR DESCRIPTION
**Pull request by diana-macrometa**
A PR to add stream workers and improve list functions.

### CODE REVIEW

File: docs/cep/query-guide/functions/list/tokenize.md [hunk 1]
```diff
@@ -27,8 +27,21 @@ list:tokenize(<OBJECT> list, <OBJECT> ...)
...
 ```sql
-@info(name = 'query1')
 list:tokenize(customList)
 ```
 
 The `list:tokenize(customList)` function takes a list named `customList`, and for each element in the list, it generates an event with the corresponding value. In this example, if `customList` contains elements (`gdn`, `IBM`, `XYZ`), the function will generate three events with value attributes `gdn`, `IBM`, and `XYZ` respectively.
+
+## Example 2
+
+```sql
```
Comment (diana-macrometa) [inline docs/cep/query-guide/functions/list/tokenize.md:37]: @koshy-thomas and @Luba89, I got this error when I tried to validate the stream worker.

Error: Error on 'T-Bkuyr63ZGDeG2XqTUmVfGF:root:_system:test' @ Line: 8. Position: 24, near 'INSERT INTO OutputStream'. Different definition same as output 'stream definition OutputStream (index int, value object)' already exist as '@Sink( type = "stream", stream = "OutputStream", @Map( type = "json"))stream definition OutputStream (index int, value string)'

**Comment by elof** [comment]
@diana-macrometa I reviewed the copy and checked that the pages are rendering correctly. If that's all you need for this review :shipit:

File: docs/cep/query-guide/functions/list/tokenize.md (hunk 1 above)
Comment (StoyanTc) [reply docs/cep/query-guide/functions/list/tokenize.md:37]: @diana-macrometa - Here the sink's value has to be `OBJECT`. Please give a try.

**Review by StoyanTc — APPROVED** [review]

File: docs/cep/query-guide/functions/list/tokenize.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/cep/query-guide/functions/list/tokenize.md:37]: Yes, that did it. Thanks!

**Pull request merged**